### ISSUE TITLE
Add Shows button inactive in Firefox 60.0.1 (64-Bit)

### ISSUE THREAD

**Issue by STX2k** [pymedusa/Medusa#4304]
I can not add TV Shows. I use the latest medusa version with nginx Reverse Proxy from wiki. The Add Show button doesnt work. 

**Branch/Commit: master** 
**OS: Ubuntu 16.05**
**What you did: Want to add an Show**
**What happened: The Add Show Button do nothing.**
**What you expected: it will add the show ;)**
**Logs: No Errors in the logs. :( But here my log:   **
```
2018-06-03 16:26:24 INFO     TORNADO :: [212cd1c] 101 GET /medusa/ws/ui (127.0.0.1) 0.49ms
2018-06-03 16:26:24 INFO     TORNADO :: [212cd1c] 304 GET /medusa/api/v2/config/main (127.0.0.1) 0.77ms
2018-06-03 16:26:23 INFO     TORNADO :: [212cd1c] 200 GET /medusa/errorlogs/viewlog/?min_level=10&log_filter=None&log_period=one_day&log_search= (127.0.0.1) 43.35ms
2018-06-03 16:26:18 INFO     TORNADO :: [212cd1c] 200 GET /medusa/addShows/sanitizeFileName?name=Final%20Space (127.0.0.1) 0.68ms
2018-06-03 16:26:18 INFO     TORNADO :: [212cd1c] 304 GET /medusa/addShows/sanitizeFileName?name=Final%20Space (127.0.0.1) 1.10ms
2018-06-03 16:26:08 INFO     TORNADO :: [212cd1c] 101 GET /medusa/ws/ui (127.0.0.1) 0.36ms
2018-06-03 16:26:08 INFO     TORNADO :: [212cd1c] 200 GET /medusa/api/v2/config/main (127.0.0.1) 1.16ms
2018-06-03 16:26:08 INFO     TORNADO :: [212cd1c] 200 GET /medusa/css/lib/vue-snotify-material.css?51755 (127.0.0.1) 1.45ms
2018-06-03 16:26:08 INFO     TORNADO :: [212cd1c] 200 GET /medusa/errorlogs/viewlog/?min_level=10&log_filter=None&log_period=one_day&log_search= (127.0.0.1) 41.90ms
2018-06-03 16:26:03 INFO     TORNADO :: [212cd1c] 200 GET /medusa/addShows/sanitizeFileName?name=Final%20Space (127.0.0.1) 0.79ms
2018-06-03 16:26:03 INFO     TORNADO :: [212cd1c] 200 GET /medusa/addShows/searchIndexersForShowName?search_term=Final%20Space&lang=de&indexer=0 (127.0.0.1) 1008.83ms
2018-06-03 16:26:03 DEBUG    Thread_2 :: [212cd1c] Using custom UI <class 'medusa.classes.AllShowsListUI'>
2018-06-03 16:26:03 DEBUG    Thread_2 :: [212cd1c] Searching for show: final space
2018-06-03 16:26:03 DEBUG    Thread_2 :: [212cd1c] Searching for Show with searchterm(s): ['Final Space'] on Indexer: TMDB
2018-06-03 16:26:02 DEBUG    Thread_2 :: [212cd1c] Using custom UI <class 'medusa.classes.AllShowsListUI'>
2018-06-03 16:26:02 DEBUG    Thread_2 :: [212cd1c] User-Agent: Medusa/0.2.3 (Linux; 4.4.0-127-generic; c0f2d1a0-6738-11e8-84f2-009c029bcbd0)
2018-06-03 16:26:02 DEBUG    Thread_2 :: [212cd1c] GET URL: http://api.tvmaze.com/search/shows?q=final%20space [Status: 200]
2018-06-03 16:26:02 DEBUG    Thread_2 :: [212cd1c] Searching for show final space
2018-06-03 16:26:02 DEBUG    Thread_2 :: [212cd1c] Searching for Show with searchterm(s): ['Final Space'] on Indexer: TVmaze
2018-06-03 16:26:02 DEBUG    Thread_2 :: [212cd1c] Using custom UI <class 'medusa.classes.AllShowsListUI'>
2018-06-03 16:26:02 DEBUG    Thread_2 :: [212cd1c] User-Agent: Medusa/0.2.3 (Linux; 4.4.0-127-generic; c0f2d1a0-6738-11e8-84f2-009c029bcbd0)
2018-06-03 16:26:02 DEBUG    Thread_2 :: [212cd1c] GET URL: https://api.thetvdb.com/search/series?name=final+space [Status: 200]
2018-06-03 16:26:02 DEBUG    Thread_2 :: [212cd1c] Searching for show: final space
2018-06-03 16:26:02 DEBUG    Thread_2 :: [212cd1c] Searching for Show with searchterm(s): ['Final Space'] on Indexer: TVDBv2
2018-06-03 16:25:52 INFO     TORNADO :: [212cd1c] 101 GET /medusa/ws/ui (127.0.0.1) 0.49ms
2018-06-03 16:25:52 INFO     TORNADO :: [212cd1c] 304 GET /medusa/api/v2/config/main (127.0.0.1) 0.82ms
2018-06-03 16:25:52 INFO     TORNADO :: [212cd1c] 200 GET /medusa/errorlogs/viewlog/?min_level=10&log_filter=None&log_period=one_day&log_search= (127.0.0.1) 43.07ms
2018-06-03 16:25:51 INFO     TORNADO :: [212cd1c] 101 GET /medusa/ws/ui (127.0.0.1) 0.40ms
2018-06-03 16:25:51 INFO     TORNADO :: [212cd1c] 200 GET /medusa/api/v2/config/main (127.0.0.1) 1.07ms
2018-06-03 16:25:50 INFO     TORNADO :: [212cd1c] 200 GET /medusa/js/home/status.js?51755 (127.0.0.1) 1.99ms
2018-06-03 16:25:50 INFO     TORNADO :: [212cd1c] 200 GET /medusa/js/home/restart.js?51755 (127.0.0.1) 2.49ms
2018-06-03 16:25:50 INFO     TORNADO :: [212cd1c] 200 GET /medusa/js/home/post-process.js?51755 (127.0.0.1) 3.03ms
2018-06-03 16:25:50 INFO     TORNADO :: [212cd1c] 200 GET /medusa/js/add-shows/recommended-shows.js?51755 (127.0.0.1) 3.37ms
2018-06-03 16:25:50 INFO     TORNADO :: [212cd1c] 200 GET /medusa/js/add-shows/popular-shows.js?51755 (127.0.0.1) 3.85ms
2018-06-03 16:25:50 INFO     TORNADO :: [212cd1c] 200 GET /medusa/js/config/index.js?51755 (127.0.0.1) 4.43ms
2018-06-03 16:25:50 INFO     TORNADO :: [212cd1c] 200 GET /medusa/js/api.js?51755 (127.0.0.1) 5.08ms
2018-06-03 16:25:50 INFO     TORNADO :: [212cd1c] 200 GET /medusa/js/lib/fix-broken-ie.js?51755 (127.0.0.1) 6.31ms
2018-06-03 16:25:50 INFO     TORNADO :: [212cd1c] 200 GET /medusa/css/lib/jquery.qtip-2.2.1.min.css?51755 (127.0.0.1) 1.32ms
2018-06-03 16:25:50 INFO     TORNADO :: [212cd1c] 200 GET /medusa/addShows/newShow/ (127.0.0.1) 4.94ms
```

**Comment by sharkykh** [comment]
60.0.1? You mean Firefox?

Anyway, most likely the same issue as #4229, which is fixed on the develop branch.

**Comment by sharkykh** [comment]
Actually just to be sure can you check the developer / inspect console on your browser for errors after you click "Add Show"?

**Comment by STX2k** [comment]
Oh damn, yes i mean Firefox - Yes i checked the Konsole, but no error, just nothing. There is only a Information: 

```
You are running Vue in development mode.
Make sure to turn on production mode when deploying for production.
See more tips at https://vuejs.org/guide/deployment.html
vue.js:8553:7

Loading local Vue since we found a window.app
newShow:1283:17

Notifications library loaded.
core.js:186:5
```

**Comment by sharkykh** [comment]
Maybe open the Network tab and then click the Add Shows button?

**Comment by STX2k** [comment]
Hey sharkykh there is also nothing. I have made a quick video (Gif ~ 100MB) https://imgur.com/a/nWrMSqa
I have no idea what i do wrong ^^

**Comment by sharkykh** [comment]
Are there any logs you can check for the reverse proxy? The request doesn't seem to get to Medusa, so mayne something in the Reverse Proxy configuration blocks it?
I don't have a setup to check this myself.

**Comment by STX2k** [comment]
Here my complete nginx config, i check nginx error logs to, but there was only one error but not related to medusa (tautulli error)

```
server {
    listen 80;
    server_name xxxx.tld www.xxxx.tld;
    return 301 https://$host$request_uri;

# ACME-challenge
    location ^~ /.well-known/acme-challenge/ {
       root /var/www/_letsencrypt;
    }
}
server {
    listen 443 ssl http2;
    server_name xxxx.tld www.xxxx.tld;
    root /var/www/xxxx.tld/organizr;
    index  index.php index.html index.htm;
    error_page 400 401 402 403 404 405 408 500 502 503 504 $scheme://$server_name/?error=$status;

    location / {
        try_files $uri $uri/ =404;
    }

    location ~ \.php$ {
        include snippets/fastcgi-php.conf;
        fastcgi_pass unix:/run/php/php7.1-fpm.sock;
    }

    location ~ /\.ht {
        deny all;
    }

    location ~ /auth-(.*) {
        internal;
        rewrite ^/auth-(.*) /api/?v1/auth&group=$1;
    }


##################################################################
######################  Error Logs  ##############################
##################################################################

    #access_log /var/log/nginx/access.log;
    error_log /var/log/nginx/error.log warn;

##################################################################
######################  SSL Certificate ##########################
##################################################################

    ssl_certificate /etc/letsencrypt/live/xxxx.tld/fullchain.pem;
    ssl_certificate_key /etc/letsencrypt/live/xxxx.tld/privkey.pem;
    ssl_trusted_certificate /etc/letsencrypt/live/xxxx.tld/fullchain.pem;
    ssl_dhparam /etc/letsencrypt/dhparam.pem;
    ssl_session_timeout 1d;
    ssl_session_cache shared:SSL:50m;
    ssl_session_tickets off;
    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
    ssl_ciphers ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:DES-CBC3-SHA:!DSS;
    ssl_prefer_server_ciphers on;
    ssl_stapling on;
    ssl_stapling_verify on;
    resolver 8.8.8.8 8.8.4.4 208.67.222.222 208.67.220.220 valid=60s;
    resolver_timeout 2s;

###################################################################
###################### Security headers ###########################
###################################################################

    add_header X-Frame-Options "SAMEORIGIN" always;
    add_header X-XSS-Protection "1; mode=block" always;
    add_header X-Content-Type-Options "nosniff" always;
    add_header Referrer-Policy "no-referrer-when-downgrade" always;
    add_header Content-Security-Policy "default-src * data: 'unsafe-eval' 'unsafe-inline'" always;
    add_header Strict-Transport-Security "max-age=31536000; includeSubDomains; preload" always;

####################################################################
###################### Services Configuration ######################
####################################################################
        location ^~ /rutorrent/ {
                auth_request /auth-1;
                proxy_pass http://127.0.0.1:8002/;
                add_header X-Frame-Options "SAMEORIGIN";
                proxy_redirect off;
                proxy_set_header Host $host;
                proxy_set_header X-Real-IP $remote_addr;
                proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        }
########################################################################################
        location /sabnzbd {
                auth_request /auth-1;
                proxy_pass http://127.0.0.1:8010;
                proxy_set_header Host $host;
                proxy_set_header X-Real-IP $remote_addr;
                proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        }
#########################################################################################
        location /medusa {
                auth_request /auth-1;
                proxy_pass http://localhost:8081;
                proxy_set_header Host $host;
                proxy_set_header X-Real-IP $remote_addr;
                proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
                proxy_set_header X-Forwarded-Host $host:443;
                proxy_set_header X-Forwarded-Server $host;
                proxy_set_header X-Forwarded-Port 443;
                proxy_set_header X-Forwarded-Proto $scheme;

        # Websocket
                proxy_http_version 1.1;
                proxy_set_header Upgrade $http_upgrade;
                proxy_set_header Connection "upgrade";
                #proxy_set_header Connection $http_connection;
                proxy_read_timeout 86400;

        }

}
```

**Comment by STX2k** [comment]
Hey, now i get an error in nginx *happy*

```
04/06/2018 18:46:53 	warn	upstream server temporarily disabled while connecting to upstream	217.225.218.182	xxx.tld	GET	/medusa/ws/ui	movy.tv	
04/06/2018 18:46:53 	error	connect() failed (111: Connection refused) while connecting to upstream	217.225.218.182	xxx.tld	GET	/medusa/ws/ui	movy.tv	
```

**Comment by sharkykh** [comment]
On the network tab gif it seems like the request isn't being received by the server.
Can you click on one of the rows and see if the url it's requesting is correct?

**Comment by STX2k** [comment]
The Post Command is empty .. nothing will send to the server :(

https://i.imgur.com/3W6tG8m.png

I have checked several configs. but changes nothing. Also check the site you have posted and edit my config. but the websocket seems to run with my config i'vd posted in upper comment, i get notification i the Browser about config changes.

**Comment by sharkykh** [comment]
If you can, try without going through the reverse proxy.

There's a chance the next version of Medusa will solve the issue,
because I updated the Javascript library that sends this request (#4232).

Other than that, unfortunately I'm out of ideas.

**Comment by sharkykh** [comment]
@STX2k We just released version 0.2.5, can you please try adding a show?

**Comment by STX2k** [comment]
@sharkykh | Yes it worked :+1:  :grinning: - Thank you very much for your help!
